gulp-tasks/wfHelper.js

```diff
@@ -49,11 +52,11 @@ function promisedExec(cmd, cwd) {
       stdOut = stdOut.trim();
       stdErr = stdErr.trim();
       if (err) {
-        gutil.log(' ', cmdLog, chalk.red('FAILED'));
```

Comment:
Why remove this? If several commands are running asynchronously, you won't know which one failed.

Reply:
Removed because it balloons logs - added back in.